web/models/owner.ex

```diff
@@ -0,0 +1,62 @@
+defmodule HexWeb.Owner do
```

Comment:
I think this should be plural if it's going to be consistent with the future `HexWeb.Releases` and `HexWeb.Packages`. Or did you have a specific reason for making it singular?